### DISCUSSION TITLE
Lesson 7: Smartcontract Lottery

### DISCUSSION THREAD

**Discussion by sundaepromix** [smartcontractkit/full-blockchain-solidity-course-py#1837] [Q&A] [answered]
**My Terminal**

Running 'scripts/deploy_lottery.py::main'...
  File "brownie/_cli/run.py", line 51, in main
    return_value, frame = run(
  File "brownie/project/scripts.py", line 110, in run
    return_value = f_locals[method_name](*args, **kwargs)
  File "./scripts/deploy_lottery.py", line 47, in main
    deploy_lottery()
  File "./scripts/deploy_lottery.py", line 8, in deploy_lottery
    account = get_account()
  File "./scripts/helpful_scripts.py", line 14, in get_account
    return accounts.add(config["wallets"]["from_key"])
  File "brownie/network/account.py", line 142, in add
    w3account = web3.eth.account.from_key(private_key)
  File "eth_utils/decorators.py", line 18, in _wrapper
    return self.method(obj, *args, **kwargs)
  File "eth_account/account.py", line 250, in from_key
    key = self._parsePrivateKey(private_key)
  File "eth_utils/decorators.py", line 18, in _wrapper
    return self.method(obj, *args, **kwargs)
  File "eth_account/account.py", line 775, in _parsePrivateKey
    return self._keys.PrivateKey(HexBytes(key))
  File "hexbytes/main.py", line 38, in __new__
    bytesval = to_bytes(val)
  File "hexbytes/_utils.py", line 17, in to_bytes
    return hexstr_to_bytes(val)
  File "hexbytes/_utils.py", line 54, in hexstr_to_bytes
    return binascii.unhexlify(ascii_hex)
Error: Non-hexadecimal digit found

**brownie-config.yaml**

dependencies:
  - smartcontractkit/chainlink-brownie-contracts@1.1.1
  - OpenZeppelin/openzeppelin-contracts@3.4.0
compiler:
  solc:
    remappings:
      - '@chainlink=smartcontractkit/chainlink-brownie-contracts@1.1.1'
      - '@openzeppelin=OpenZeppelin/openzeppelin-contracts@3.4.0'
dotenv: .env
networks:
  default: development
  development: 
    keyHash: "0x2ed0feb3e7fd2022120aa84fab1945545a9f2ffc9076fd6156fa96eaff4c1311"
    fee: 100000000000000000
  goerli: 
    vrf_coordinator: "0x2bce784e69d2Ff36c71edcB9F88358dB0DfB55b4"
    eth_usd_price_feed: "0xD4a33860578De61DBAbDc8BFdb98FD742fA7028e"
    link_token: "0x326C977E6efc84E512bB9C30f76E30c160eD06FB"
    keyHash: "0x0476f9a745b61ea5c0ab224d3a6e4c99f0b02fce4da01143a4f70aa80ae76e8a"
    fee: 100000000000000000
    verify: True
  mainnet-fork:
   eth_usd_price_feed: "0x5f4eC3Df9cbd43714FE2740f5E3616155c5b8419"
   verify: False
wallets: 
  from_key: $(PRIVATE_KEY)

**Comment by sundaepromix** [comment]
please i need help

**Comment by cromewar** [comment, marked as answer]
Hello @sundaepromix 
On your brownie config yaml you are using the PRIVATE_KEY with parenthesis "()" and you must use "{}" with curly brackets brackets. there error is saying that it could not found the key as the format you are trying to use is incorrect.

**Reply by sundaepromix** [reply]
i now now see my error.
Thanks so much. i really appreciate
